controllers/C_Prescription.class.php

```diff
@@ -159,7 +159,7 @@ function list_action($id, $sort = "")
                     // (array_filter removes empty items)
                     $rxcui_list = implode("+", array_filter($nameList));
                     // Do not urlencode the $rxcui_list, since this breaks the + items
-                    $data = file_get_contents("https://rxnav.nlm.nih.gov/REST/interaction/list.json?rxcuis=" . $rxcui_list);
+                    $data = file_get_contents_curl("https://rxnav.nlm.nih.gov/REST/interaction/list.json?rxcuis=" . $rxcui_list);
```

Comment:
This is a nice one to start with since we can test it rather easily

Reply:
Yes, we can easily adapt it as necessary in the future.

Reply:
We already have guzzle available, so could try this here instead to try out guzzle (I think this will work :) ):
```
$client = new GuzzleHttp\Client();
$response = $client->get("https://rxnav.nlm.nih.gov/REST/interaction/list.json?rxcuis=".$rxcui_list);
```

Reply:
To test this, rxnorms full has to be installed. For some reason supported external loads no longer default support rxnorm import.